snap/info.go

```diff
@@ -527,9 +534,22 @@ func (app *AppInfo) ServiceFile() string {
 	return filepath.Join(dirs.SnapServicesDir, app.ServiceName())
 }
 
-// ServiceSocketFile returns the systemd socket file path for the daemon app.
-func (app *AppInfo) ServiceSocketFile() string {
-	return filepath.Join(dirs.SnapServicesDir, app.SecurityTag()+".socket")
+// ServiceSocketFiles returns the list of  systemd socket file paths for the daemon app.
+func (app *AppInfo) ServiceSocketFiles() []string {
```

Comment:
We probably don't need that one. We can easily iterate as:
```
for name, socket := range app.Sockets {
    fmt.Println(name, socket.File())
}
```
See below as well.